### PR TITLE
Repeat message

### PR DESCRIPTION
**Pull request by joshcwinton**
Description of PR here...

### Frontend:
- Create function to post message to backend
### Backend:
-

### CODE REVIEW

File: frontend/src/components/Chat.jsx [hunk 1]
```diff
@@ -15,7 +36,7 @@ class Chat extends Component {
   submitMessage = messageString => {
     // on submitting the ChatInput form, send the message, add it to the list and reset the input
     const message = { name: this.state.name, message: messageString };
```
Comment (joshcwinton) [inline frontend/src/components/Chat.jsx]: can we change message.message to message.text? @JasperCheung

Comment (JasperCheung) [reply frontend/src/components/Chat.jsx]: yes, that's makes it more reable i think.

File: frontend/src/components/Chat.jsx [hunk 2]
```diff
@@ -14,8 +35,8 @@ class Chat extends Component {
 
   submitMessage = messageString => {
     // on submitting the ChatInput form, send the message, add it to the list and reset the input
-    const message = { name: this.state.name, message: messageString };
-    this.addMessage(message);
+    const message = { name: this.state.name, text: messageString };
```
Comment (JasperCheung) [inline frontend/src/components/Chat.jsx:38]: Cool, i like this better

File: frontend/src/components/ChatMessage.jsx [hunk 3]
```diff
@@ -2,5 +2,5 @@ import React from 'react';
 
 export default ({ name, message }) =>
   <p>
-    <strong>{name}</strong> <em>{message}</em>
+    <strong>{name}</strong>: <em>{message}</em>
```
Comment (JasperCheung) [inline frontend/src/components/ChatMessage.jsx:5]: colon D

File: frontend/src/components/Chat.jsx [hunk 4]
```diff
@@ -1,11 +1,32 @@
...
+  }
+
+  // Send message to backend then print it to console
+  sendMessage = (message) => {
+    // add message to chat history
+    this.addMessage(message);
+
+    // send message to backend
+    axios.post('http://0.0.0.0:5555/sentences', {
+        sass: message.text
```
Comment (JasperCheung) [inline frontend/src/components/Chat.jsx:20]: this `sass` thing has to change for the future. We should decide on a way to send responses that will be consistent down the line. 

also, i don't like the route name being `sentences`, lets change it to something like `repeat-messages` or `echo`

@svhnstorage @joshcwinton @donaldchen34

**Review by JasperCheung — CHANGES_REQUESTED** [review]
Looks good. 

regarding npm: 
- remove the `package.json` and `package-lock.json` from the root directory and add `axios` to the packages that exist in the `frontend` dir.

File: frontend/src/components/Chat.jsx (hunk 4 above)
Comment (joshcwinton) [reply frontend/src/components/Chat.jsx:20]: agreed, i think most things should end up being variations of `message` and `text`

Comment (joshcwinton) [reply frontend/src/components/Chat.jsx:20]: created an issue, @svhnstorage @donaldchen34, can one of you pick this up from the project board?

**Comment by joshcwinton** [comment]
@JasperCheung moved package files

**Review by JasperCheung — APPROVED** [review]
YASSSSS KING!

**Pull request merged**